x/supplier/client/cli/query_proof_test.go

```diff
@@ -0,0 +1,145 @@
+package cli_test
+
+// TODO_UPNEXT(@Olshansk): Add these tests back in after updating how Proofs are stored on-chain.
```

Comment:
Am I understanding this correctly and/or wdyt about phrasing like so?:

```suggestion
// TODO_UPNEXT(@Olshansk): Add these tests back in after merging on-chain Proof persistence.
```